pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/manager/controller_manager_config.go

```diff
@@ -64,14 +64,14 @@ webhook:
 leaderElection:
   leaderElect: true
   resourceName: {{ hashFNV .Repo }}.{{ .Domain }}
-# leaderElectionReleaseOnCancel defines if the leader should step down volume
+# leaderElectionReleaseOnCancel defines if the leader should step down voluntarily
 # when the Manager ends. This requires the binary to immediately end when the
 # Manager is stopped, otherwise, this setting is unsafe. Setting this significantly
-# speeds up voluntary leader transitions as the new leader don't have to wait
-# LeaseDuration time first.
+# speeds up voluntary leader transitions as the new leader doesn't have to wait
+# the LeaseDuration time first.
 # In the default scaffold provided, the program ends immediately after
-# the manager stops, so would be fine to enable this option. However,
-# if you are doing or is intended to do any operation such as perform cleanups
+# the manager stops, so it would be fine to enable this option. However,
+# if you are doing, or are intending to do, any operation such as perform cleanups
```

Comment:
kustomize/v1 is deprecated and is no longer used unless you scaffold a project using go/v3 explicity
It will be removed asap after the next release
So, can you please revert those changes.

Reply:
Sure thing!